src/index.ts

```diff
@@ -10,18 +10,37 @@
  */
 
 import * as ts from 'typescript';
+let argv = require('yargs')
```

Comment:
I started with `import`, but it seems they use this syntax for getting in those extra configuration methods before the actual method extraction. At that point I chose to just follow their way, but open to suggestions.